console/console-init/ui/src/modules/project/components/ProjectDatatoolbarToggleGroup/ProjectDataToolbarToggleGroup.tsx

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import {
+  SelectOptionObject,
+  DataToolbarToggleGroup,
+  DataToolbarGroup,
+  DataToolbarFilter,
+  InputGroup,
+  Button,
+  DataToolbarItem,
+  ButtonVariant,
+  DataToolbarChip,
+  DataToolbarChipGroup,
+  DropdownPosition,
+  Badge
+} from "@patternfly/react-core";
+import { FilterIcon, SearchIcon } from "@patternfly/react-icons";
+import {
+  DropdownWithToggle,
+  TypeAheadSelect,
+  DropdownWithBulkSelect
+} from "components";
+import { typeOptions, filterMenuItems } from "modules/project/utils";
+
+export interface IProjectToolbarToggleGroupProps {
+  totalRecords: number;
+  filterSelected?: string;
+  nameSelected?: string;
+  nameInput?: string;
+  namespaceSelected?: string;
+  namespaceInput?: string;
+  nameOptions?: any[];
+  namespaceOptions?: any[];
+  typeSelected?: string | null;
+  selectedNames: Array<{ value: string; isExact: boolean }>;
+  selectedNamespaces: Array<{ value: string; isExact: boolean }>;
+  onFilterSelect: (value: string) => void;
+  onNameSelect: (e: any, selection: SelectOptionObject) => void;
```

Comment:
can we please try event name as onSelectName,onSelectNamespace, onClearName,onClearNamspace, onSelectType
eventName+postfit=onSelect+Name/Namespace.